src/test/Components/ImagesTable/ImagesTable.test.js

```diff
@@ -309,7 +313,7 @@ const mockStatus = {
   },
   '4873fd0f-1851-4b9f-b4fe-4639fce90794': {
     image_status: {
-      status: 'success',
+      status: 'expired',
```

Comment:
https://github.com/osbuild/image-builder/blob/891b573d509e9ad77e93b9e3ee58f63141322ec9/internal/v1/api.yaml#L371

`expired` is not a state that image builder can return, see the above list.

Reply:
Thank you! Understood, the `expiring` and `expired` statuses exist only in the front end and don't map with api 1:1. I've updated the tests to accommodate that.